### ISSUE TITLE
feature req: Ability to enlarge some elements in navigation UI

### ISSUE THREAD

**Issue by cptMikky** [osmandapp/Osmand#1605]
Some elements (mostly in the "right panel") may be more important than others in different use cases. 

For instance when driving with active navigation, Current Speed, Speed Limit or ETA are informations that the driver might find more useful than others and would like to have them enlarged.

It would be nice to be able to set an "icon" in the right panel enlarged per navigation use case, just like it is alreadz possible to show them, disable them or hide them under the expanding arrow.

cheers!

**Comment by vshcherb** [comment]
Yes we are thinking in the same direction and have mockup for this, but I think it will not be done by 2.2 release so I would say 2.3 will be there

**Comment by kaueraal** [comment]
Just a bit more input: I'd prefer to be able to adjust each widgets size individually. For different sized widgets resolving issue #1608 would allow to put the larger widgets at e.g. the top.

**Comment by Al-Hun** [comment]
I was wondering about something similirar : I'm runner and use osmand to register my tracks. It's very difficult while running to read the indications, such as time/distance elapsed, speed...
My wish would be the ability by pressing a key to zoom for parameterized time whole right panel.
So it needed to the ability of allocating standards actions to an available key (may be pressing the key to choose it ?). Ideally it would work with this kind of additive button (which plug in the earphone connector) : 
![image](https://cloud.githubusercontent.com/assets/14762270/10017118/2ab47d98-612c-11e5-8f1e-62338d94d9c4.png)

Hopefully, thanks,
Alain.

**Comment by szszoke** [comment]
I am using OsmAnd for biking and I would also appreciate to be able to enlarge some widgets, like my current speed. It is quite hard to read as it is now.

**Comment by gubrist** [comment]
![Screenshot_20230803_161244_OsmAnd+](https://github.com/osmandapp/OsmAnd/assets/22197180/56633acd-0167-497c-b8bb-381b8840b538)

It would be helpfull to enlarge de MAP SCALE.

Because:
I use myy bicycle in summer with (sun)glasses which makes it hard to read the map scale and assess distances swiftly.
